src/Components/PageSelection/PageSelection.js

```diff
@@ -0,0 +1,16 @@
+import { useState } from 'react'
+import Home from '../Home'
+import EditProduct from '../EditProduct'
+
+const PageSelection = (props) => {
+    const [selectedPage, setSelectedPage] = useState(props.pages.HOME)
+
+    return (
+        <div>
```

Comment:
we could use React Fragment here to avoid having another <div> if we don't need it.

Reply:
done, thanks